### DISCUSSION TITLE
hashicropkrug

### DISCUSSION THREAD

**Discussion by MCJunga** [tedilabs/fastcampus-devops#5] [General]
hashicropkrug 페이스 그룹이나 슬랙은 사라진 듯 합니다.

**Comment by posquit0** [comment]
@MCJunga 다음 링크에서 확인하실 수 있습니다!

페이스북: https://www.facebook.com/groups/hashicorpkrug
슬랙: https://slack.opencontainer.co.kr 내 #hashicorp-ug 채널 이용

**Comment by MinHyeong-Lee** [comment]
강의자료 내부에 설명된 링크에 오타가 있더라구요

**Reply by posquit0** [reply]
앗 그랬군요.. 감사합니다! ㅠㅠ
